index.html

```diff
@@ -2,17 +2,17 @@
 <html>
 
 <head>
-  <meta charset="utf-8" />
+  <meta charset="UTF-8" />
   <title>Re:Earth</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap" rel="stylesheet">
 </head>
 
 <body>
   <div id="root"></div>
+  <script type="module" src="/src/main.tsx"></script>
```

Comment:
What is this for?

Reply:
Read https://vitejs.dev/guide/#index-html-and-project-root